app/models/application_record.rb

```diff
@@ -20,4 +20,9 @@ def self.replace_scope(name, body, &block)
     singleton_class.undef_method name
     scope name, body, &block
   end
+
+  def self.vacuum_table(full_with_lock: true)
+    opts = 'FULL' if full_with_lock
+    connection.exec_query("VACUUM #{opts} #{connection.quote_table_name(table_name)}")
```

Comment:
"VACUUM FULL" might be helpful for recovering space in other tables without prolonged RDS downtime